openslides/utils/settings.py.tpl

```diff
@@ -26,6 +26,7 @@ INSTALLED_PLUGINS += (
 
 # Built-in plugins:
 #    'tests.example_data_generator',
+    'django_extensions',
```

Comment:
Can you explain why we need this?

If we need it, it shoul'd be in in openslides.global_settings and not in the template.

Reply:
The explanation to this is in my comment before.